apps/web/src/services/my-page/my-account/user-account/modules/MultiFactorAuthentication.vue

```diff
@@ -0,0 +1,171 @@
+import VerifyButton from '@/common/modules/button/verify-button/VerifyButton.vue';
+
+import UserAccountModuleContainer
+    from '@/services/my-page/my-account/user-account/modules/UserAccountModuleContainer.vue';
+
+// Currently, only email is supported.
+const contextMenuItems = [
+    { label: i18n.t('IDENTITY.USER.MFA.EMAIL'), name: 'email' },
+];
+
+const state = reactive({
+    disable: false,
+    isVerified: computed(() => store.state.user.mfa !== undefined),
+    userId: computed(() => store.state.user.userId),
+    domainId: computed(() => store.state.domain.domainId),
+});
+
+const handleChangeEnableToggle = () => {
+    state.disable = !state.disable;
+};
+
+const {
+    forms: {
+        email,
+    },
+    setForm,
+    invalidState,
+    invalidTexts,
+} = useFormValidator({
+    email: store.state.user.email,
+}, {
+    email(value: string) { return !emailValidator(value) ? '' : i18n.t('IDENTITY.USER.FORM.EMAIL_INVALID'); },
+});
+</script>
+
+<template>
+    <user-account-module-container
+        class="multi-factor-authentication-wrapper"
+    >
+        <template #headline>
+            <div class="headline-wrapper">
+                <p class="form-title">
+                    {{ $t('IDENTITY.USER.MFA.TITLE') }}
+                </p>
+                <div class="verify-status-wrapper">
+                    <div v-if="state.isVerified"
+                         class="verified"
+                    >
+                        <p-i name="ic_verified"
+                             height="1rem"
+                             width="1rem"
+                             class="verified-icon"
+                             color="#60B731"
+                        />
+                        <span>
+                            {{ $t('IDENTITY.USER.MFA.VERIFIED') }}
+                        </span>
+                    </div>
+                    <span v-else
+                          class="not-verified"
+                    >
+                        {{ $t('IDENTITY.USER.MFA.NOT_VERIFIED') }}
+                    </span>
+                </div>
+            </div>
+        </template>
+        <div class="enable-toggle">
+            <p-field-title class="toggle-title">
+                {{ $t('IDENTITY.USER.MFA.ENABLE_MFA') }}
+            </p-field-title>
+            <p-toggle-button :value="!state.disable"
+                             show-state-text
+                             position="left"
+                             @change-toggle="handleChangeEnableToggle"
+            />
+        </div>
+        <form v-if="!state.disable"
+              class="form"
+              onsubmit="return false"
+        >
+            <p-field-group
+                :label="$t('IDENTITY.USER.MFA.TYPE')"
+                required
+                class="field-group"
+            >
+                <p-select-dropdown :menu="contextMenuItems"
+                                   :selected="contextMenuItems[0].name"
```

Comment:
suggestion: It might be better to use the `selected` variable even when there is only one menu currently😅